README.rst

```diff
@@ -97,22 +97,17 @@ familiar with the `PyAnsys Developer's Guide`_.
 Test
 ====
 
-.. note::
-
-   The Docker container referenced in the first option is not yet publicly available.
-
```

Comment:
License specification not needed anymore because it is already set when starting the container